<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreatePatternOperation</class>
 <widget class="QWidget" name="FormCreatePatternOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание образца операции</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя образца операции</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldPatternName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип операции</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPatternType">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Трассировка</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTrase">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип времени</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTypeTime">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение времени</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldTimeValue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Закон</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLaw">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начало интервала</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Конец интервала</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldBeginInt">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldEndInt">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить релевантный ресурс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить релевантный ресурс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать образец операции</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1051</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1049</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
